### PR TITLE
[CHANGELOG] diem-core-v1.3.0 release

### PR DESCRIPTION
**Pull request by sausagee**
## Motivation
Prepping for v1.3 release

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

yes

## Test Plan
None other than CI.

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.3.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Changelog
+
+## [Unreleased](https://github.com/diem/diem/tree/HEAD)
+
+[Full Changelog](https://github.com/diem/diem/compare/diem-core-v1.2.0...HEAD)
+
+**Fixed bugs:**
+
+- \[Bug\] Got BadProxy on release 1.2.0 when running cli [\#8297](https://github.com/diem/diem/issues/8297)
```
Comment (bob-wilson) [inline CHANGELOG/CHANGELOG-1.3.md:9]: Did you see my comment on the internal doc? It sounds like at least some of these were due to configuration changes and should not really be listed as "fixed" bugs.

Comment (sausagee) [reply CHANGELOG/CHANGELOG-1.3.md:9]: Dang it, i didn't see it (the contrast of the highlight is nearly indistinguishable to my eyes) until you pointed out here.   

I agree.  These aren't really bug fixes as there is no PR attached. I will remove them.

**Review by bob-wilson — APPROVED** [review]
I only reviewed the details of sections that I know something about, but those look pretty good now.

**Comment by sausagee** [comment]
fixed lint and squashed

**Review by bob-wilson — DISMISSED** [review]

**Pull request closed without merging**